css/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  background: palevioletred;
```

Comment:
Your background color doesn't follow the template one, as you were making a copy I'd suggest that you try to make it as similar as possible of the original one